agent/consul/leader.go

```diff
@@ -1201,6 +1201,7 @@ func (s *Server) handleAliveMember(member serf.Member) error {
 			},
 			Meta: map[string]string{
 				"non_voter":             strconv.FormatBool(member.Tags["nonvoter"] == "1"),
```

Comment:
Same comment addition as before,
```suggestion
				// DEPRECATED - remove nonvoter in favor of read_replica in a future version of consul
				"non_voter":             strconv.FormatBool(member.Tags["nonvoter"] == "1"),
```